commons/src/test/scala/dagr/commons/io/IoTest.scala

```diff
@@ -132,4 +149,8 @@ class IoTest extends UnitSpec {
     val grandchild = PathUtil.pathTo(dir.toAbsolutePath.toString, "grand/child")
     Io.findFirstExtentParent(grandchild).get.toAbsolutePath.toString shouldBe dir.toAbsolutePath.toString
   }
+
+  it should "return None if no parent was found" in {
+    Io.findFirstExtentParent(PathUtil.pathTo("/")) shouldBe 'empty
```

Comment:
This is a strange test case - this was a private method that was only ever invoked on paths that's don't exist.  I'm not sure how I feel about calling it on paths that do exist, but because they are the root, their parent doesn't.